diff --git a/node_modules/@react-native-picker/picker/android/build.gradle b/node_modules/@react-native-picker/picker/android/build.gradle
index 155b3a8..ce6841c 100644
--- a/node_modules/@react-native-picker/picker/android/build.gradle
+++ b/node_modules/@react-native-picker/picker/android/build.gradle
@@ -75,9 +75,11 @@ if (isNewArchitectureEnabled()) {
 
 android {
     def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION
-  if (agpVersion.tokenize('.')[0].toInteger() >= 7) {
     namespace "com.reactnativecommunity.picker"
-  }
+
+    buildFeatures {
+        buildConfig = true
+    }
 
   compileSdkVersion safeExtGet('compileSdkVersion', 31)
 
